new-log-viewer/src/services/LogExportManager.ts

```diff
@@ -0,0 +1,70 @@
+import {downloadBlob} from "../utils/file";
+
+
+const EXPORT_LOG_PROGRESS_INITIALIZATION = 0;
+const EXPORT_LOG_PROGRESS_COMPLETE = 1;
```

Comment:
I think INITIALIZATION & COMPLETE are more descriptive in LOG_PROGESS. lmk about your thoughts

Reply:
They're not so intuitive to me:

* EXPORT_LOG_PROGRESS_INITIALIZATION sounds like it could be a flag to enable whether we enable "log progress init".
* INITIALIZATION and COMPLETE don't seem like two ends of a spectrum to me.
* If EXPORT_LOG_PROGRESS_MIN and MAX don't sound as clear to you, how about EXPORT_LOG_PROGRESS_VALUE_MIN and MAX?

@junhaoliao, what do you think?

Reply:
Both makes sense. 
How about keeping the const names and adding docs for each to explain which is the min value and which is the max?

Reply:
> how about EXPORT_LOG_PROGRESS_VALUE_MIN and MAX?

Sounds better.